schema.graphql

```diff
@@ -419,6 +420,43 @@ type TransferManager @entity {
   updatedBlock: Block!
 }
 
+type StatType @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset!
+  opType: String!
+  claimType: String
```

Comment:
Does SubQuery support enums with the new version? I remember Diego submitted a PR upstream for it but not sure if it got merged in.

If it does, then I think it would be nice to have this as an enum type.

If not OK to leave as string though.

Reply:
I checked and Yes enums are working. I'll create a separate task to add enum support wherever possible in subquery